api/capi/src/nnstreamer-capi-util.c

```diff
@@ -621,9 +621,6 @@ ml_tensors_data_create (const ml_tensors_info_h info, ml_tensors_data_h * data)
   if (status != ML_ERROR_NONE) {
     return status;
   }
-  if (!_data) {
-    return ML_ERROR_STREAMS_PIPE;
-  }
 
   for (i = 0; i < _data->num_tensors; i++) {
     _data->tensors[i].tensor = g_malloc0 (_data->tensors[i].size);
```

Comment:
I think `g_malloc0` cannot fail. If you wanna handle failure cases, how about using `g_try_malloc0`? Or, you may remove `ML_ERROR_OUT_OF_MEMORY` as well. In fact, size 0 can return NULL, but I think it's not out-of-memory but close to invalid arguments.

Reply:
Well, internally most APIs and functions in nnstreamer does not use g_try_malloc() now.
As you said, g_malloc internally uses g_error() which aborts the process - always fatal - if failed to allocate mem.
How about adding another PR for changing all allocation (g_try_malloc0 ot g_try_new0)?

Reply:
Yes, my comment was not related to this PR :)